sdk/storage/Azure.Storage.Blobs.Batching/src/Azure.Storage.Blobs.Batching.csproj

```diff
@@ -4,6 +4,7 @@
   </PropertyGroup>
   <PropertyGroup>
     <AssemblyTitle>Microsoft Azure.Storage.Blobs.Batching client library</AssemblyTitle>
+    <Version>12.0.0-preview.5</Version>
```

Comment:
how does this play with https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Directory.Build.props#L13

Reply:
It overrides it. ```<Version />``` is normally a calculated value combining ```<VersionPrefix/ >``` and ```<VersionSuffix />``` but this overrides that calculation and makes it explicit.

/cc @weshaggard

Reply:
We will have to move the version property into each of the storage libraries, at least the ones we are going to ship as GA vs preview one set will have to be in the project files directly.

Reply:
If we are putting in all the projects then we should remove the shared version to avoid any confusion.

Reply:
I've removed the version from ```Directory.Build.props```.